app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -19,10 +19,16 @@ import '../request_handling/request_handler.dart';
 import '../service/buildbucket.dart';
 
 /// List of Github supported repos.
-const Set<String> supportedRepos = <String>{'engine', 'flutter', 'cocoon'};
+const Set<String> kSupportedRepos = <String>{'engine', 'flutter', 'cocoon'};
```

Comment:
Why the k?

Reply:
Just following the guideline at https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k. I know it's not required for this cocoon repo, but it sounds like a good idea.